README.md

```diff
@@ -225,6 +227,80 @@ When using backreferences, there is a couple of things to keep in mind:
 - `__via__` is surrounded with two underscores; you can read this query out loud like this: _"get posts that link to author via the author field"_
 
 
+## Helpers
+
+`cf-graphql` comes with included helpers. These are used inside of [the demo application](https://github.com/contentful-labs/cf-graphql/tree/master/demo).
```

Comment:
I'd say "...with included helpers _that help you with `cf-graphql` integration_." I don't feel like it's important to mention they are used in the demo since they are generic and can be used anywhere.